wp-content/mu-plugins/pantheon.php

```diff
@@ -19,4 +19,7 @@
 	if ( ! defined('RETURN_TO_PANTHEON_BUTTON') || RETURN_TO_PANTHEON_BUTTON ) {
 		require_once 'pantheon/pantheon-login-form-mods.php';
 	}
+	if ( ! defined('FS_METHOD') || FS_METHOD ) {
```

Comment:
I'm not sure we need the `FS_METHOD` check there. This would be true if there was no defined constant OR if there WAS a defined constant (which would be incorrect).

Also, Nit: 
Typically, WP coding standards are to have spaces inside all parentheticals, so this would be:

```php
if ( ! defined( 'FS_METHOD' ) ) {
    define( 'FS_METHOD', 'direct' );
}
```

Reply:
Ah, I was trying to mimic the formatting and behaviors of the rest of the file as closely as possible, in hopes that that would be respecting coding style. 😅

I'll force-push the changes.. since I've been asked to make sure the PR is a single commit and not a stack of commits.

Reply:
Yeah, the context is different for those things. 😁

Reply:
Yeah.. in my haste I didn't think it through.. 😳   Thank goodness for code review